provisioner/provisioner.go

```diff
@@ -231,7 +235,8 @@ func (p *Provisioner) validateNodeAffinityRules() error {
 
 	if len(nodeList) == 0 {
 		return errors.Errorf(
-			"No matching nodes found for given affinity rules (%s)",
+			"%s (%s)",
+			NodeAffinityRulesMismatchEvent,
```

Comment:
nit: this can be merge in one line

Reply:
Yeah, missed it I will try to not repeat this again